test/unit/gateways/cyber_source_rest_test.rb

```diff
@@ -148,6 +176,49 @@ def test_add_shipping_address
     assert_equal '4158880000', address[:phoneNumber]
   end
 
+  def test_authorize_apple_pay_visa
+    stub_comms do
+      @gateway.authorize(100, @apple_pay, @options)
+    end.check_request do |_endpoint, data, _headers|
+      request = JSON.parse(data)
+      assert_equal '001', request['paymentInformation']['tokenizedCard']['type']
+      assert_equal '1', request['paymentInformation']['tokenizedCard']['transactionType']
+      assert_equal nil, request['paymentInformation']['tokenizedCard']['requestorId']
```

Comment:
💬 Comment:

For this case an the others `assert_nil` is a more direct way of this assertion